src/lib_json/json_writer.cpp

```diff
@@ -872,11 +872,11 @@ struct CommentStyle {
 };
 
 struct BuiltStyledStreamWriter : public StreamWriter {
-  BuiltStyledStreamWriter(JSONCPP_STRING const& indentation,
+  BuiltStyledStreamWriter(JSONCPP_STRING  indentation,
                           CommentStyle::Enum cs,
-                          JSONCPP_STRING const& colonSymbol,
-                          JSONCPP_STRING const& nullSymbol,
-                          JSONCPP_STRING const& endingLineFeedSymbol,
+                          JSONCPP_STRING  colonSymbol,
```

Comment:
everything being edited in this neighborhood has two spaces after JSONCPP_STRING for some reason.
Maybe a scripting error in generating the change?